client/components/Home.jsx

```diff
@@ -3,33 +3,43 @@ import {connect} from 'react-redux'
 
 import OrderList from './OrderList'
 import AddToOrder from './AddToOrder'
-import {requestCurrentOrder, requestUsers, orderComplete} from '../actions'
+import {
+  requestCurrentOrder,
+  requestUsers,
+  orderComplete,
+  deleteItemById
+} from '../actions'
 
 class Home extends React.Component {
   constructor (props) {
     super(props)
     this.markComplete = this.markComplete.bind(this)
+    this.deleteItem = this.deleteItem.bind(this)
+  }
+
+  componentDidMount () {
+    this.props.dispatch(requestCurrentOrder())
+    this.props.dispatch(requestUsers())
   }
 
   markComplete () {
     this.props.dispatch(orderComplete(this.props.orderId))
   }
 
-  componentDidMount () {
-    this.props.dispatch(requestCurrentOrder())
-    this.props.dispatch(requestUsers())
+  deleteItem (id) {
+    this.props.dispatch(deleteItemById(id))
   }
 
   render () {
     const orders = this.props.orders || []
     return (
       <div className='order-container'>
         <h2>Current Order</h2>
-        <OrderList orders={orders} />
+        <OrderList orders={orders} onClickFn={this.deleteItem} />
         <div className="completed">
           <button onClick={this.markComplete}>Mark as Complete</button>
         </div>
-        <AddToOrder />
+        <AddToOrder/>
```

Comment:
I think we're including the space on most self-closing elements/components, so you can put this back.